### ISSUE TITLE
[SFML] Including spine-sfml.h causes 'Polygon': redefinition error on windows

### ISSUE THREAD

**Issue by maxxboehme** [EsotericSoftware/spine-runtimes#1058]
When including spine-sfml.h I get the error 'Polygon': redefinition in wingdi.h. I assume this is because I am including some SFML headers that must be including windows.h which is bringing in that file. spine-sfml.h should be refactored to be able to remove "SPINE_SHORT_NAMES" so these name collisions don't happen.

There appears to be a workaround though described on [StackOverflow](https://stackoverflow.com/questions/38693570/redefinition-error-in-c).
Basically add #define NOGDI before including SFML headers.

**Comment by badlogic** [comment]
Thanks for reporting. I've fixed this in the 3.7-beta branch.
